kogito-editors-js/packages/boxed-expression-component/src/components/Table/EditableCell.tsx

```diff
@@ -140,10 +140,19 @@ export const EditableCell: React.FunctionComponent<EditableCellProps> = React.me
 
     const onTextAreaChange = useCallback(
       (event) => {
+        const newValue: string = event.target.value || "";
+
+        if (textarea.current && (newValue.includes("\t") || newValue.includes("\n"))) {
```

Comment:
Can we, please, assign a boolean const to that condition? Something like `isPastedValue` or `IsComposedValue`

Reply:
Fixed! Thanks :)